ci/requirements/environment.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
+  - asv
```

Comment:
wouldn't it be better to put `asv` in the `extra-specs` parameter of the micromamba action? You might have to set a custom cache key for `cache-env` to work properly, though.

Reply:
Will the same cached environment be used across different workflows? For example does `ubuntu-latest py3.10` use the same cache as `Benchmark`? 

If no I think I understand what you mean; because you can't avoid the downloads on different workflows anyway. But then I don't get why a new `cache-env` is needed?

If yes then I'm probably misunderstanding something but wouldn't that create another cached environment with more or less same packages?  
So now we have `xarray-tests` folder that's lets say 10mb large. Now we create another` xarray-tests-asv` that contains `asv` as well which maybe makes it maybe this folder maybe 11mb large. So now there's 21 mb stored somewhere in the cloud when it could've been 11mb in total only.

Reply:
That depends on the cache key, which by default is a combination of date and file hash, if I remember correctly.

The main CI jobs explicitly set a custom cache key, which is necessary because we specify the python version in `extra-specs`. As a consequence, the benchmark job will use a different cache file at the moment.

Once we do set the same key they would indeed use the same cache, so if the benchmark would always run we'd gain something by including it in the main environment. However, we currently don't run the benchmark on every commit (I think?), so we'd only get that 11MB cache once in a few days (e.g. after a merge), while cache invalidation happens once every day.